### PR TITLE
refactor: remove deprecated service-store protocols

### PR DESCRIPTION
**Pull request by hardikmaheshwari**
## Summary
Remove deprecated service-store protocols

## How did you test this change?

- [ ] Test(s) added
- [ ] Manual testing (please provide screenshots/recordings)
- [X] No testing (please provide an explanation) - This part of code is deprecated and should not be used

### CODE REVIEW

**Comment by akphi** [comment]
@hardikmaheshwari are we waiting for some engine change for this one?

**Comment by hardikmaheshwari** [comment]
> @hardikmaheshwari are we waiting for some engine change for this one?

Not any changes but I am in process of confirming if we are good removing this and doesn't break anyone.

**Comment by akphi** [comment]
Gotcha! Thanks for remember doing this!! 😃

**Comment by hardikmaheshwari** [comment]
@akphi We are good to release this now

**Review by akphi — APPROVED** [review]

File: .changeset/happy-candles-tan.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-extension-external-store-service': minor
```
Comment (akphi) [inline .changeset/happy-candles-tan.md:2]:
```suggestion
'@finos/legend-extension-external-store-service': major
```

**Review by akphi — APPROVED** [review]

**Pull request merged**